src/tyr/route_serializer_osrm.cc

```diff
@@ -664,11 +664,14 @@ json::ArrayPtr intersections(const valhalla::DirectionsLeg::Maneuver& maneuver,
         auto indications = json::array({});
         uint16_t mask = turn_lane.directions_mask();
 
+        // Ensure not to duplicate "uturn" indications
+        bool had_uturn = false;
+
         // TODO make map for lane mask to osrm indication string
         // reverse (left u-turn)
-        if ((mask & kTurnLaneReverse) &&
-            (maneuver.type() == DirectionsLeg_Maneuver_Type_kUturnLeft)) {
+        if (mask & kTurnLaneReverse) {
```

Comment:
This was a fix for a missing "uturn" indication for intersections on the previous maneuver, we were serializing an empty list `"indications": []"`. 

@gknisely caught this on turn_lane_routes.txt:14.

Reply:
/cc @dgearhart @mandeepsandhu can you check my logic here?